testing/CMakeLists.txt

```diff
@@ -96,6 +96,41 @@ target_include_directories(xfpga-static PUBLIC
   PRIVATE ${OPAE_SDK_SOURCE}/libopae/plugins/xfpga
   PRIVATE ${OPAE_SDK_SOURCE}/libopae)
 
+add_library(opae-c-ase-static STATIC
+        ${OPAE_SDK_SOURCE}/ase/sw/tstamp_ops.c
+        ${OPAE_SDK_SOURCE}/ase/sw/ase_ops.c
+        ${OPAE_SDK_SOURCE}/ase/sw/ase_strings.c
+        ${OPAE_SDK_SOURCE}/ase/sw/ase_host_memory.c
+        ${OPAE_SDK_SOURCE}/ase/sw/app_backend.c
+        ${OPAE_SDK_SOURCE}/ase/sw/mqueue_ops.c
+        ${OPAE_SDK_SOURCE}/ase/sw/error_report.c
+        ${OPAE_SDK_SOURCE}/ase/api/src/common.c
+        ${OPAE_SDK_SOURCE}/ase/api/src/buffer.c
+        ${OPAE_SDK_SOURCE}/ase/api/src/close.c
+        ${OPAE_SDK_SOURCE}/ase/api/src/enum.c
+        ${OPAE_SDK_SOURCE}/ase/api/src/event.c
+        ${OPAE_SDK_SOURCE}/ase/api/src/manage.c
+        ${OPAE_SDK_SOURCE}/ase/api/src/reconf.c
+        ${OPAE_SDK_SOURCE}/ase/api/src/mmio.c
+        ${OPAE_SDK_SOURCE}/ase/api/src/open.c
+        ${OPAE_SDK_SOURCE}/ase/api/src/umsg.c
+        ${OPAE_SDK_SOURCE}/ase/api/src/version.c
+        ${OPAE_SDK_SOURCE}/ase/api/src/wsid_list.c
+        ${OPAE_SDK_SOURCE}/ase/api/src/error.c)
+
+target_include_directories(opae-c-ase-static PUBLIC
+    $<BUILD_INTERFACE:${OPAE_INCLUDE_DIR}>
+    $<BUILD_INTERFACE:${CMAKE_BINARY_DIR}/include>
+    $<INSTALL_INTERFACE:include>
+    PRIVATE ${OPAE_SDK_SOURCE}/ase/api/src
+    PRIVATE ${OPAE_SDK_SOURCE}/ase/sw)
```

Comment:
Could we consider moving the ASE source directory under libopae/plugins in the repo?

Reply:
I would move it for the JIRA task of adding ASE plugin. This task is only for porting the ASE tests to new test framework.

Reply:
sure